.github/scripts/extract-review.py

```
@@ -0,0 +1,82 @@
+                depth -= 1
+                if depth == 0:
+                    try:
+                        obj = json.loads(raw[start:i + 1])
+                    except Exception:
+                        obj = None
+                    break
+
+if not obj or obj.get('verdict') not in VERDICTS:
+    sys.exit(1)
+
+lines = ['## AI 审核参考：%s' % obj['verdict'], '', '### ✅ 验证过做对的部分']
```

Ruff (0.16.2)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)